spec/requests/api/v2/users/generate_and_send_magic_link_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+describe "PATCH /api/v2/users/generate_and_send_magic_link", type: :request do
+  let(:user)    { create(:user) }
+  let(:payload) { { email: user.email } }
+
+  let(:token) do
+    JWT.encode payload,
+               Settings.authentication.secret,
+               Settings.authentication.algorithm
+  end
+
+  let(:credentials) do
+    ActionController::HttpAuthentication::Token.encode_credentials(token)
+  end
+
+  def perform_request
+    patch(
+      generate_and_send_magic_link_api_v2_users_path,
+      headers: { "HTTP_AUTHORIZATION" => credentials },
+      params: {},
+    )
+  end
+
+  context "when unauthenticated" do
+    let(:payload) { { email: "foo@bar" } }
+
+    before do
+      perform_request
+    end
+
+    subject { response }
+
+    xit { should have_http_status(:ok) }
```

Comment:
These are being skipped

Reply:
That's intentional, for now. There's a ticket in Trello to finish this work but I can't login to Trello at the moment.

Reply:
👍